README.md

```diff
@@ -106,6 +120,22 @@ aggregators:
     hosts : "agg-31"
     group : l3-agg
 
+samplers:
+  - names       : *sampler-hosts
+    hosts       : *sampler-hosts
+    ports       : *sampler-ports
+    group       : samplers
```

Comment:
If you do it like I described above, you don't need the group name.

Reply:
The group name is to easily group similar configurations into one dict object, but you are correct, we could just as easily organize by the "name". I thought the "group" attribute made it slightly more readable than comparing a list of host "names", but I'm fine with doing away with it.